docs/production.md

```diff
@@ -52,7 +52,7 @@ Start the installation for CentOS 7:
 
     vagrant up centos7-katello-3.4
 
-This will create a libvirt based virtual machine running the Katello server on CentOS.
+This will create a libvirt based virtual machine running the Katello server on CentOS. 
```

Comment:
Please leave this part out
```suggestion
This will create a libvirt based virtual machine running the Katello server on CentOS.
```

Reply:
I'm not sure I incorporated this correctly. I think perhaps you meant "Please don't leave this part out". I'm not sure if the "Suggested change" is referring to my change or you are making the suggestion. The two diffs look the same.

Reply:
@ekohl is there anything else that is needed on this?

Reply:
fixed in https://github.com/theforeman/forklift/pull/1185/commits/60f2326b96424c4d1faf6c644e389b4d49282844